visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -408,17 +408,13 @@ describe("codeMapMouseEventService", () => {
 				event = { button: ClickType.RightClick, clientX: 0, clientY: 1 }
 			})
 
-			it("should $broadcast a building-right-clicked event with data", () => {
+			it("should $broadcast a building-right-clicked event ", () => {
```

Comment:
```suggestion
			it("should $broadcast a building-right-clicked event", () => {
```

It is not obvious that the click is indeed a right click in this case. Is it possible to verify that as well?

Reply:
we fire a button:ClickType.RightClick before each test, shouldnt that cover it?